### PR TITLE
First connect, then ask for env name - closes #1744

### LINKED ISSUE #1744: Ops Tools - Change flow: ask user for environment name *after* establishing connection
This Issue reflects the change in usability flow for connecting to an ops tools instance, in accordance with the requested flow to connect to an IBM cloud instance (future work)..

The order of events will now be:
- ask the user to select add environment method
- if ops tools, ask for URL, api key and API secret
- try to establish connection
  - ask for CA certificate chain if needed, and try again
- save CA certificate file and store key and secret
- ask user to chose a name for the environment
- ask the user to select which nodes to import

### PR DESCRIPTION
**Pull request by lquintai**
Signed-off-by: Leonor Quintais <lquintai@uk.ibm.com>

### CODE REVIEW

File: packages/blockchain-extension/extension/util/InternalUtil.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { ExtensionUtil } from './ExtensionUtil';
+
+/*
+ * This file will hold generic util functions used by the extension.
+ */
+export class InternalUtil {
+
+    public static getCoreNodeModule(moduleName: string): any {
+        try {
+          return ExtensionUtil.getModuleAsar(moduleName);
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/util/InternalUtil.ts]: these functions might as well go in here too

File: packages/blockchain-extension/extension/util/InternalUtil.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+*/
+
+'use strict';
+
+import { ExtensionUtil } from './ExtensionUtil';
+
+/*
+ * This file will hold generic util functions used by the extension.
+ */
+export class InternalUtil {
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/util/InternalUtil.ts]: not sure i like the name but not sure what to call it

Comment (lquintai) [reply packages/blockchain-extension/extension/util/InternalUtil.ts]: Changing to ModuleUtil

File: packages/blockchain-extension/extension/util/ModuleUtil.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+'use strict';
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+/*
+ * This file will hold generic util functions used by the extension.
+ */
+export class ModuleUtil {
+    public static getModuleAsar(moduleName: string): any {
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/util/ModuleUtil.ts]: this should be private

File: packages/blockchain-extension/extension/util/ModuleUtil.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        return require(path.join(vscode.env.appRoot, 'node_modules.asar', moduleName));
+    }
+
+    public static getModule(moduleName: string): any {
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/util/ModuleUtil.ts]: this should also be private

**Pull request merged**